.gitignore

```diff
@@ -108,3 +108,5 @@ refix
 .vscode
 .kitchen
 .emacs.desktop*
+tests/topotests/.DS_Store
```

Comment:
# Ignore Mac DS_Store files
change `tests/topotests/.DS_Store` to `.DS_Store` to ignore all macOS DS_Store files

Reply:
Modified the .gitignore file.